scipy/signal/tests/test_filter_design.py

```diff
@@ -3787,14 +3787,28 @@ def test_frequency_response(self):
     def test_iir_symmetry(self):
         b, a = gammatone(440, 'iir', fs=24000)
         z, p, k = tf2zpk(b, a)
-        sorted(z) == pytest.approx(sorted(z.conj()))
-        sorted(p) == pytest.approx(sorted(p.conj()))
-        k == pytest.approx(np.real(k))
+        assert_array_equal(sorted(z), sorted(z.conj()))
+        assert_array_equal(sorted(p), sorted(p.conj()))
+        assert_equal(k, np.real(k))
 
         assert_(issubclass(b.dtype.type, np.floating))
         assert_(issubclass(a.dtype.type, np.floating))
 
-    # Verify IIR filter coefficients with MATLAB's answers
+    # Verify FIR filter coefficients with the paper's
+    # Mathematica implementation
```

Comment:
If it was wrong the first time you computed it in MATLAB, it's worth adding a note here about the difference you found so that the next person who cross-checks isn't confused by the difference they find

Reply:
The paper does not have a MATLAB implementation of FIR gammatone filters, only IIR gammatone filters. So I cross-checked my answers with the paper's Mathematica formulas instead. I have included the page references in the main function. Do you want me to include the references in the unit test comments as well?

Reply:
Ahh okay, I misunderstood. I think this is fine as is!